aconfigure.ac

```diff
@@ -2021,7 +2023,7 @@ AC_ARG_ENABLE(ssl,
     		    fi
 
         	    AC_CHECK_PROGS(PKG_CONFIG,
-                       $host-pkg-config pkg-config "python pkgconfig.py",
+                       $host_cpu-$host_os-pkg-config pkg-config "python pkgconfig.py",
```

Comment:
Googling around (such as:
https://autotools.info/pkgconfig/cross-compiling.html)
, `$host-pkg-config` seems to be commonly used as well, so better not remove it.

Reply:
Good point, will keep that, too.

Reply:
done